relation_engine/taxa/ncbi/loaders/ncbi_taxa_delta_loader.py

```diff
@@ -1,117 +1,66 @@
-        'if --user is included and --pwd-file is omitted a password prompt will be presented.')
-    parser.add_argument(
-        '--load-registry-collection',
-        required=True,
-        help='the name of the ArangoDB collection where the load will be registered. ' +
-        'This is typically the same collection for all delta loaded data.')
-    parser.add_argument(
-        '--node-collection',
-        required=True,
-        help='the name of the ArangoDB collection into which taxa nodes will be loaded')
-    parser.add_argument(
-        '--edge-collection',
-        required=True,
-        help='the name of the ArangoDB collection into which taxa edges will be loaded')
-    parser.add_argument(
-        '--merge-edge-collection',
-        required=True,
-        help='the name of the ArangoDB collection into which merge edges will be loaded')
-    parser.add_argument(
-        '--load-version',
-        required=True,
-        help='the version of this load. This version will be added to a field in the nodes and ' +
-             'edges and will be used as part of the _key field.')
-    parser.add_argument(
-        '--load-timestamp',
-        type=int,
-        required=True,
-        help='the timestamp to be applied to the load, in unix epoch milliseconds. Any nodes ' +
-             'or edges created in this load will start to exist with this time stamp. ' +
-             'NOTE: the user is responsible for ensuring this timestamp is greater than any ' +
-             'other timestamps previously used to load data into the NCBI taxonomy DB.')
-    parser.add_argument(
-        '--release-timestamp',
-        type=int,
-        required=True,
-        help='the timestamp, in unix epoch milliseconds, when the data was released ' +
-        'at the source.')
-
-    return parser.parse_args()
+    parser.add_argument('--config', required=True,
+                        help='the path to the loader configuration file. NOTE: the config '
+                        + 'file will need to be updated for each consecutive load; it is not '
+                        + 'static.')
+    a = parser.parse_args()
+    with open(a.config, 'rb') as c:
+        return DeltaLoaderConfig(c, [_INPUT_DIRECTORY], require_merge_collection=True)
 
 
 def main():
-    a = parse_args()
-    nodes = os.path.join(a.dir, NODES_IN_FILE)
-    names = os.path.join(a.dir, NAMES_IN_FILE)
-    merged = os.path.join(a.dir, MERGED_IN_FILE)
-    client = ArangoClient(hosts=a.arango_url)
-    if a.user:
-        if a.pwd_file:
-            with open(a.pwd_file) as pwd_file:
-                pwd = pwd_file.read().strip()
-        else:
-            pwd = getpass.getpass()
-        db = client.db(a.database, a.user, pwd, verify=True)
+    cfg = get_config()
+    rootdir = cfg.inputs[_INPUT_DIRECTORY]
+    nodes = rootdir / NODES_IN_FILE
+    names = rootdir / NAMES_IN_FILE
+    merged = rootdir / MERGED_IN_FILE
```

Comment:
as neat as this operator shorthand is, it doesn't seem quite as readable as an explicit path join

Reply:
also might result in weird errors if rootdir or NODES_IN_FILE are somehow strings or some other unexpected type

Reply:
> as neat as this operator shorthand is, it doesn't seem quite as readable as an explicit path join

here I have to disagree; it's literally the path separator. What could be more explicit than that :P

> also might result in weird errors if rootdir or NODES_IN_FILE are somehow strings or some other unexpected type

The configuration class guarantees that rootdir is a `Path`. `NODES_IN_FILE` is a string - the `Path` class handles that case.

Reply:
> What could be more explicit than that  

Using the function that the operator overload hides behind what is elsewhere division 😇 
Not a big deal but, I'd maintain, a bit unfriendly to the passing reader

Reply:
I mean it's not really any different than `+` meaning addition in context A and concatenation in context B